sroka/api/s3_connection/s3_connection_api.py

```diff
@@ -74,3 +75,33 @@ def s3_download_data(s3_filename, prefix=False, output_file=None, sep=','):
         data.to_csv(output_file, sep=sep)
 
     return data
+
+
+def s3_upload_data(data, bucket, path, sep=','):
+    key_id = config.get_value('aws', 'aws_access_key_id')
+    access_key = config.get_value('aws', 'aws_secret_access_key')
+    session = boto3.Session(
+        aws_access_key_id=key_id,
+        aws_secret_access_key=access_key
+    )
+
+    try:
+        if type(data) == pd.core.frame.DataFrame:
+            csv_buffer = StringIO()
```

Comment:
Could be created as just `buffer = StringIO()` before try-except and used in both cases